jdaviz/configs/specviz2d/plugins/cross_dispersion_profile/cross_dispersion_profile.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <j-tray-plugin
+    plugin_key="Cross Dispersion Profile"
+    :description="docs_description"
+    :uses_active_status="uses_active_status"
+    :keep_active.sync="keep_active"
+    @plugin-ping="plugin_ping($event)"
+    :popout_button="popout_button"
+    :scroll_to.sync="scroll_to">
+
+    <plugin-dataset-select
+      :items="dataset_items"
+      :selected.sync="dataset_selected"
+      :show_if_single_entry="false"
+      label="Data"
+      hint="Select the data for photometry."
+    />
+
+    <v-row>
+      <v-col cols="9" style="padding: 0px">
+        <v-slider
+          :min="0"
+          :max="max_pix"
+          v-model.number="pixel"
+          density="compact"
+          hint="Pixel on spectral axis."
+          persistent-hint
+        >
+      </v-col>
+
+      <v-col cols="3" style="padding: 7px">
+        <v-text-field
+          v-model.number="pixel"
+          type="number"
+          style="width: 60px"
+          label="Pixel"
+          hide-details
+          dense
+          single-line
+        />
+      </v-col>
+    </v-row>
+
+    <div v-if="!use_full_width">
+      <v-row>
+        <v-col cols="9" style="padding: 0px">
+          <v-slider
+            :min="0"
+            :max="max_y_pix"
+            v-model.number="y_pixel"
+            density="compact"
+            hint="Center of profile on cross-dispersion axis."
+            persistent-hint
+          >
+        </v-col>
+
+        <v-col cols="3" style="padding: 7px">
+          <v-text-field
+            v-model.number="y_pixel"
+            type="number"
+            style="width: 60px"
+            label="Pixel"
+            hide-details
+            dense
+            single-line
+          />
+        </v-col>
+    </v-row>
+
+      <v-row>
+        <v-col cols="9" style="padding: 0px">
+          <v-slider
+            :min="0"
+            :max="max_y_pix"
+            v-model.number="width"
+            density="compact"
+            hint="Width in cross-dispersion axis, in pixels."
+            persistent-hint
+          >
+        </v-col>
+
+        <v-col cols="3" style="padding: 7px">
+          <v-text-field
+            v-model.number="width"
+            type="number"
+            style="width: 60px"
+            label="Pixel"
+            hide-details
+            dense
+            single-line
+          />
+        </v-col>
+      </v-row>
+    </div>
+
+    <plugin-switch
+      :value.sync="use_full_width"
+      label="Use full cross-dispersion width."
+      style="font-size: 10px;"
+    />

```

Comment:
it makes sense to me to have this on top of the things that appear/disappear based on its value, but can also run by @Jenneh

Reply:
Hmm, I do like the sliders all being together, but I see Kyle's point (and it might be nice to not have the toggle move away from your mouse when you click it). I think I'm leaning towards having it on top but curious what Jenn would say.

Reply:
i reordered the elements, the plot is now on top, and the sliders now appear under the toggle